serverless/onnx/WongKinYiu/yolov7/nuclio/function-gpu.yaml

```diff
@@ -95,7 +95,7 @@ spec:
   handler: main:handler
   eventTimeout: 30s
   build:
-    image: cvat.onnx.wongkinyiu.yolov7
+    image: cvat.onnx.wongkinyiu.yolov7:latest-gpu
```

Comment:
Shall we update other images as ``latest-cpu`` or just ``latest`` for consistency?

Reply:
The default _is_ `latest`. I don't think there's any need to set it explicitly.